.github/actions/opence-conda-package-required-pr/action.yml

```diff
@@ -0,0 +1,16 @@
+name: Open-CE Conda Package Build and Install Tests
+
+author: "Open-CE"
+
+description: "Required test for Open-CE Repo."
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install dependencies
+      shell: bash -l {0}
+      run: |
+        open_ce_path="$GITHUB_ACTION_PATH/../../../"
+        sh ${open_ce_path}/packaging_scripts/create_conda_package.sh
+        conda install -y open-ce -c file://${open_ce_path}/open-ce-conda-pkg
+        conda list open-ce
```

Comment:
We probably shouldn't make an action for this, since it's only going to be used in this repo. Instead, you can just add this run step directly to the workflow yaml file.

Reply:
Yeah, earlier I'd added this in the workflow itself. But it was failing to access open-ce root level directory for some reason. I'll retry that change.

Reply:
Adding it back to workflow and fixing the paths worked. Thanks @bethune-bryant for your review and inputs.

Reply:
Looks good, thanks!